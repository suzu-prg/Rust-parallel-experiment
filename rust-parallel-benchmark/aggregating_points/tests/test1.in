10 10 50216456 1613000515 258023955 253187976 1025204373 1980624267 -794749222 1060537562 -2090212007 -1580307007 926222703 -970125279 -732076489 1606622185 -994062978 -482382084 1582102724 -186468678 257810638 -1220681940 1126695541 -1703237593 1334002607 1634461004 -1726950747 -1601034730 -1930677942 529785122 -1741304189 -1280333227 -1385122001 -170655869 -393833348 1665366440 -1216131972 -1423161531 -187927805 -1980213461 1245339531 1728836598 -1004302738 -1422708966 1822918417 550030288 134659461 -2094998952 1848397601 434271529 -1623933708 1673807442 920986623 428348277 1328122774 1092808833 -1977758231 581228950 -1868990426 -1604471135 442105169 -301612041 30258817 1537500441 1848354570 -741298259 1046056803 -118572444 -1981162290 -1848556037 -1021056582 -1100274675 842882895 126454772 -1873980326 1369378561 -1105499980 -1591501237 1489839307 1201416552 719740039 1469144982 311647168 1913734530 -17267679 1842337860 -1992276362 117488279 -729168685 -308077827 944669263 -1614084471 1921351545 -235352415 -89243113 1784234126 -1358429345 956108089 1239458146 -2047355543 -992971905 -859491128 