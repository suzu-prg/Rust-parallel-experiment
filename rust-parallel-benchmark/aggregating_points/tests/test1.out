-2094998952 -1848556037 -1423161531 -1004302738 -393833348 30258817 434271529 956108089 1334002607 1673807442 